.github/workflows/sdk-generation.yaml

```diff
@@ -631,7 +674,7 @@ jobs:
           MAVEN_USERNAME: ${{ secrets.ossrh_username }}
           MAVEN_PASSWORD: ${{ secrets.ossrh_password }}
           ORG_GRADLE_PROJECT_signingKey: ${{ secrets.java_gpg_secret_key }}
-          ORG_GRADLE_PROJECT_signingPassphrase: ${{ secrets.java_gpg_passphrase }}
+          ORG_GRADLE_PROJECT_signingPassphrase: ${{ secrets.java_passphrase }}
```

Comment:
I dont think this is supposed to be changed either

Reply:
this must've gotten overwritten by a rebase; sorry